2021-05-09 12:49:33,544 INFO    MainThread:14994 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_LOG_MODEL
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_WATCH
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_setup.py:_flush():69] setting env: {'silent': 'true'}
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_setup.py:_flush():69] [34m[1mwandb[0m: Currently logged in as: [33mdlrgy22[0m (use `wandb login --relogin` to force relogin)
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/mycode/code/wandb/run-20210509_124933-3ujfk1ft/logs/debug.log
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/mycode/code/wandb/run-20210509_124933-3ujfk1ft/logs/debug-internal.log
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_init.py:init():369] calling init triggers
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 12:49:33,545 INFO    MainThread:14994 [wandb_init.py:init():418] starting backend
2021-05-09 12:49:33,545 INFO    MainThread:14994 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 12:49:33,631 INFO    MainThread:14994 [backend.py:ensure_launched():123] starting backend process...
2021-05-09 12:49:33,726 INFO    MainThread:14994 [backend.py:ensure_launched():128] started backend process with pid: 15167
2021-05-09 12:49:33,728 INFO    MainThread:14994 [wandb_init.py:init():423] backend started and connected
2021-05-09 12:49:33,730 INFO    MainThread:14994 [wandb_init.py:init():465] updated telemetry
2021-05-09 12:49:33,731 INFO    MainThread:14994 [wandb_init.py:init():484] communicating current version
2021-05-09 12:49:34,580 INFO    MainThread:14994 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-09 12:49:34,580 INFO    MainThread:14994 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 12:49:34,802 INFO    MainThread:14994 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 12:49:37,544 INFO    MainThread:14994 [wandb_run.py:_console_start():1554] atexit reg
2021-05-09 12:49:37,544 INFO    MainThread:14994 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.REDIRECT
2021-05-09 12:49:37,545 INFO    MainThread:14994 [wandb_run.py:_redirect():1406] Redirecting console.
2021-05-09 12:49:37,546 INFO    MainThread:14994 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-09 12:49:37,546 INFO    MainThread:14994 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 13:14:42,387 INFO    MainThread:14994 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 0
2021-05-09 13:14:42,389 INFO    MainThread:14994 [wandb_run.py:_restore():1496] restore
2021-05-09 13:14:44,595 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-05-09 13:14:45,417 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4695
}

2021-05-09 13:14:45,819 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4568
  total_bytes: 4932
}

2021-05-09 13:14:45,920 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4568
  total_bytes: 4932
}

2021-05-09 13:14:46,023 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

2021-05-09 13:14:46,124 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

2021-05-09 13:14:46,225 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

2021-05-09 13:14:46,326 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

2021-05-09 13:14:46,427 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

2021-05-09 13:14:46,528 INFO    MainThread:14994 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4932
  total_bytes: 4932
}

